Do not build SIM Access Profile (sap) plugin

The sap plugin is not enabled by default,
unlikely useful nowadays, and causes a warning
on daemon startup due to missed D-Bus policy in
/etc/dbus-1/system.d/bluetooth.conf
See also: <https://github.com/bluez/bluez/issues/441>

(Closes: #803265)

--- bluez/debian/rules.orig	2024-01-03 09:39:10.381835631 +0000
+++ bluez/debian/rules	2024-01-03 09:41:34.515448901 +0000
@@ -19,7 +19,6 @@
 	--enable-library \
 	--enable-test \
 	--enable-nfc \
-	--enable-sap \
 	--enable-monitor \
 	--enable-udev \
 	--enable-obex \
